File: hrms/src/main/java/com/obss/hrms/response/GetJobSeekerResponse.java
package com.obss.hrms.response;

import com.obss.hrms.entity.PersonalSkill;

import java.time.LocalDate;
import java.util.List;

public record GetJobSeekerResponse(
       String id,
       String firstname,
       String lastName,
       String email,
       LocalDate birthDay,
       String description,
       Boolean inBlackList,
       List<PersonalSkill> personalSkillList
) {
}

File: hrmsFe/src/app/model/getJobSeekerResponse.ts
import {GetPersonalSkillResponse} from "./getPersonalSkillResponse";

export interface GetJobSeekerResponse{

  id: string,

  firstname: string,

  lastName: string,

  email: string,

  birthDay: Date, //backendden localdate gelıyor hangı turle tutabılecegıne bak

  description: string,

  inBlackList: boolean,

  personalSkillList: GetPersonalSkillResponse[]

}
